manifests/params.pp

```diff
@@ -43,6 +40,14 @@
       $libgalera_location           = '/usr/lib64/galera/libgalera_smm.so'
       $additional_packages          = 'rsync'
     }
+    $osr_array = split($::operatingsystemrelease,'[\/\.]')
```

Comment:
that's right, but operatingsystemmajrelease is only available with Facter >= 1.7
I can change that, if you want it.
This code is a partial copy of https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/version.pp#L17